src/redux/books/books.js

```diff
@@ -1,30 +1,17 @@
-import { createAsyncThunk } from '@reduxjs/toolkit';
-
 // Actions
 const ADD = 'bookstore-react/booksReducer/ADD';
 const REMOVE = 'bookstore-react/booksReducer/REMOVE';
-const GET = 'bookstore-react/booksReducer/GET';
-
-// API URL
-const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/xKgONp1vYVZJzmgq5zMK/books';
 
 // Reducer
 export default function booksReducer(state = [], action) {
   switch (action.type) {
-    case `${ADD}/fulfilled`:
-      return state.concat(action.meta.arg);
-    case `${REMOVE}/fulfilled`:
-      return state.filter((book) => book.item_id !== action.meta.arg);
-    case `${GET}/fulfilled`:
-      return Object.keys(action.payload).map((key) => {
-        const { title, author, category } = action.payload[key][0];
-        return {
-          item_id: key,
-          title,
-          author,
-          category,
-        };
-      });
+    case ADD:
```

Comment:
- [ ] Note that for this project the purpose is to connect your app to an API and according to the project requirements, it is required to use either [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) to make API calls. But this requirement is missing 😢 Please make sure to fix that part in order to make your app works as expected 👍